application/src/main/java/org/togetherjava/tjbot/commands/basic/PingCommand.java

```diff
@@ -24,6 +30,17 @@ public PingCommand() {
      */
     @Override
     public void onSlashCommand(SlashCommandInteractionEvent event) {
-        event.reply("Pong!").queue();
+        // event.reply("Pong!").queue();
+
+        event
+            .replyModal(Modal.create(generateComponentId("AAaadw2341f"), "Yes.")
+                .addActionRow(TextInput.create("text", "text", TextInputStyle.PARAGRAPH).build())
+                .build())
+            .queue();
+    }
+
+    @Override
+    public void onModalInteraction(final ModalInteractionEvent event, final List<String> args) {
+        event.reply("Lets go!! " + event.getValue("text").getAsString()).queue();
```

Comment:
this was for testing right? this should go now

Reply:
Yeah, during my name change to onButtonClick I missed this.

Reply:
thanks for testing it though 👌